tests/tox.ini

```diff
@@ -0,0 +1,73 @@
+ Copyright 2023 Canonical Ltd.
```

Comment:
Since we don't have an integration tests right now, I'd simply remove this file as well as the pyproject.toml